metricbeat/module/vsphere/network/_meta/fields.yml

```diff
@@ -0,0 +1,48 @@
+- name: network
+  type: group
+  release: beta
+  description: >
+    Network-related information.
+  fields:
+    - name: accessible
+      type: boolean
+      description: >
+        Indicates whether at least one host is configured to provide this network.
+    - name: config.status
+      type: keyword
+      description: >
+        Indicates whether the system has detected a configuration issue.
+    - name: host
+      type: group
+      fields:
+        - name: names
+          type: keyword
+          description: >
+            Names of the hosts connected to this network.
+        - name: count
+          type: long
+          description: >
+            Number of hosts connected to this network.
+    - name: name
+      type: keyword
+      description: >
+        Name of the network.
+    - name: status
+      type: keyword
+      description: >
+        General health of the Network.
+    - name: type
+      type: keyword
+      description: >
+        Type of the network.
```

Comment:
Lets mention example of different types of network.
Eg: Standard, DistributedVirtualport

Reply:
Sure